include/core/shape.h

```diff
@@ -18,6 +18,21 @@
 
 #include <deal.II/base/auto_derivative_function.h>
 #include <deal.II/base/function.h>
+
+#include <deal.II/grid/manifold.h>
+#include <deal.II/grid/manifold_lib.h>
+
+#ifdef DEAL_II_WITH_OPENCASCADE
+#  include <deal.II/opencascade/manifold_lib.h>
+#  include <deal.II/opencascade/utilities.h>
```

Comment:
I think those two includes work if DEAL_II_WITH_OPENCASCADE=False but I am not sure.